source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -57,72 +58,108 @@ struct SuccessResponse {
   ResponsePtr response_;
 };
 
-std::vector<Matchers::LowerCaseStringMatcherPtr>
-createLowerCaseMatchers(const envoy::type::matcher::v3::ListStringMatcher& list) {
-  std::vector<Matchers::LowerCaseStringMatcherPtr> matchers;
+envoy::type::matcher::v3::StringMatcher
+lowercaseStringMatcher(const envoy::type::matcher::v3::StringMatcher& matcher) {
```

Comment:
Per your other comment I would just use the case insensitive matcher for this case?